clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7203,6 +7203,33 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
                    options::OPT_fno_ptrauth_intrinsics, false))
     CmdArgs.push_back("-fptrauth-intrinsics");
 
+  if (Args.hasFlag(options::OPT_fptrauth_calls, options::OPT_fno_ptrauth_calls,
```

Comment:
This should be added in Clang::AddAArch64TargetArgs and use addOptInFlag. See 6ff61914a1ef2c05702ba73cf31ffdeed59c1740

Reply:
Thanks! Fixed in 68bf805e9edb108dd64037ef8ed7d588ece66513